poetry.lock

```diff
@@ -0,0 +1,4686 @@
+[[package]]
```

Comment:
I thought you agreed to remove this file?

Reply:
Yes, and I did. I'm not sure how it made it into the second commit though. Removing in a min!